packages/fxa-auth-server/lib/routes/support.js

```diff
@@ -45,7 +45,10 @@ module.exports = (log, db, config, customs, zendeskClient) => {
         validate: {
           payload: isA.object().keys({
             topic: isA.string().required(),
-            subject: isA.string().optional(),
+            subject: isA
+              .string()
+              .allow('')
```

Comment:
This, or otherwise the client need to remove the `subject` property altogether from the payload when it is an empty string.

Reply:
Have you spoken to julia about removing the empty subject as an option? It seems preferable to use the "general" subject for these cases

Reply:
It's a little confusing there's a topic and a subject.  This is the subject text input, which is optional per the mock-ups.  I think you might be thinking of the topic, which is from the drop down.  That isn't optional.